difftest/dpi_t1rocketemu/src/dramsim3-config.ini

```diff
@@ -0,0 +1,68 @@
+# DDR4_8Gb_x8_3200 from DRAMsim3 upstream
```

Comment:
How about placing this file in a better place, it should be a configuration. cc @Avimitin

Reply:
The planned argument for setting DRAMsim3 settings has three values:
- +dramsim3_cfg = yes: using the embedded setting, which is this one
- +dramsim3_cfg = no: disable DRAMsim3, use trivial memory latency model
- +dramsim3_cfg = <path>: Use an external DRAMsim3 setting.

The embedded configuration here are just a quality-of-life feature, and makes CI easier to use.